ee/debug/checkups/version.go

```diff
@@ -26,17 +40,52 @@ func (c *Version) ExtraFileName() string {
 }
 
 func (c *Version) Status() Status {
-	return Informational
+	return c.status
 }
 
 func (c *Version) Summary() string {
-	return fmt.Sprintf("launcher_version %s", version.Version().Version)
+	return c.summary
 }
 
 func (c *Version) Data() any {
+	firstRunTIme, err := getTimeFromStore(c.k.LauncherHistoryStore(), "first_recorded_run_time")
+	if err != nil {
+		firstRunTIme = "unknown"
+	}
+
+	firstRunVersionStr := "unknown"
```

Comment:
nit- when you were debugging an issue last week you could immediately tell there was a build issue based on the default string version being `"unknown"` (set from packaging). I wonder if there's value to making this default something different, either empty string or the actual error we encountered collecting the value- so that we couldn't falsely interpret this as the same build issue if it is actually just not set or if some other issue creeps into here

Reply:
good catch, updated to just use blank string for both if not found